outputs.tf

```diff
@@ -1,84 +1,79 @@
   description = "The AutoScaling Group ID"
-  value       = "${module.autoscale_group.autoscaling_group_id}"
+  value       = module.autoscale_group.autoscaling_group_id
 }
 
 output "autoscaling_group_name" {
   description = "The AutoScaling Group name"
-  value       = "${module.autoscale_group.autoscaling_group_name}"
+  value       = module.autoscale_group.autoscaling_group_name
 }
 
 output "autoscaling_group_arn" {
   description = "ARN of the AutoScaling Group"
-  value       = "${module.autoscale_group.autoscaling_group_arn}"
+  value       = module.autoscale_group.autoscaling_group_arn
 }
 
 output "autoscaling_group_min_size" {
   description = "The minimum size of the AutoScaling Group"
-  value       = "${module.autoscale_group.autoscaling_group_min_size}"
+  value       = module.autoscale_group.autoscaling_group_min_size
 }
 
 output "autoscaling_group_max_size" {
   description = "The maximum size of the AutoScaling Group"
-  value       = "${module.autoscale_group.autoscaling_group_max_size}"
+  value       = module.autoscale_group.autoscaling_group_max_size
 }
 
 output "autoscaling_group_desired_capacity" {
   description = "The number of Amazon EC2 instances that should be running in the group"
-  value       = "${module.autoscale_group.autoscaling_group_desired_capacity}"
+  value       = module.autoscale_group.autoscaling_group_desired_capacity
 }
 
 output "autoscaling_group_default_cooldown" {
   description = "Time between a scaling activity and the succeeding scaling activity"
-  value       = "${module.autoscale_group.autoscaling_group_default_cooldown}"
+  value       = module.autoscale_group.autoscaling_group_default_cooldown
 }
 
 output "autoscaling_group_health_check_grace_period" {
   description = "Time after instance comes into service before checking health"
-  value       = "${module.autoscale_group.autoscaling_group_health_check_grace_period}"
+  value       = module.autoscale_group.autoscaling_group_health_check_grace_period
 }
 
 output "autoscaling_group_health_check_type" {
   description = "`EC2` or `ELB`. Controls how health checking is done"
-  value       = "${module.autoscale_group.autoscaling_group_health_check_type}"
+  value       = module.autoscale_group.autoscaling_group_health_check_type
 }
 
 output "security_group_id" {
   description = "ID of the worker nodes Security Group"
-  value       = "${join("", aws_security_group.default.*.id)}"
+  value       = join("", aws_security_group.default.*.id)
 }
 
 output "security_group_arn" {
   description = "ARN of the worker nodes Security Group"
-  value       = "${join("", aws_security_group.default.*.arn)}"
+  value       = join("", aws_security_group.default.*.arn)
 }
 
 output "security_group_name" {
   description = "Name of the worker nodes Security Group"
-  value       = "${join("", aws_security_group.default.*.name)}"
+  value       = join("", aws_security_group.default.*.name)
 }
 
-output "worker_role_arn" {
+output "workers_role_arn" {
```

Comment:
Could we note this in the PR please? I depend on this output of the module :)

Reply:
updated the PR description.
Renaming those outputs is not strictly necessary, but it reflects that the cluster has many workers. And since the new TF 0.12 module has many other changes, you will need to update it anyway